library/common/network/BUILD

```diff
@@ -76,3 +76,23 @@ envoy_cc_library(
         "@envoy//source/common/network:socket_interface_lib",
     ],
 )
+
+cc_library(
+    name = "ios_platform_verifier",
+    srcs = select({
+        "@envoy//bazel:apple": ["ios_platform_verifier.cc"],
+        "//conditions:default": [],
+    }),
+    hdrs = select({
+        "@envoy//bazel:apple": ["ios_platform_verifier.h"],
+        "//conditions:default": [],
+    }),
+    deps = select({
+        "@envoy//bazel:apple": [
+            "//library/common/extensions/cert_validator/platform_bridge:c_types_lib",
+            "//library/common:envoy_main_interface_lib",
+            "@envoy//bazel:boringssl",
+        ],
+        "//conditions:default": [],
+    }),
+)
```

Comment:
Can you add a trailing newline to match unix conventions?

Reply:
Done. Not sure how that crept back :/ (Also not sure why clang-format didn't fix it. But c'est la vie)